src/main/java/io/strimzi/kafka/bridge/KafkaBridgeProducer.java

```diff
@@ -84,7 +53,7 @@ protected void handleClose() {
      * @param record Kafka record to send
      * @return a CompletionStage bringing the metadata
      */
-    protected CompletionStage<RecordMetadata> send(ProducerRecord<K, V> record) {
+    public CompletionStage<RecordMetadata> send(ProducerRecord<K, V> record) {
```

Comment:
`Producer#send` can block. That's not a problem for how this `KafkaBridgeProducer#send` is called in the `HttpSourceBridgeEndpoint`, but maybe it's worth mentioning in the doc?